### PR TITLE
Top pvp page [Issue #89] and various other fixes along the way

### LINKED ISSUE #89: Make Top 10 pvp page
Sorted by most wins top to bottom

### PR DESCRIPTION
**Pull request by S1lentcodes**
closes #89

### CODE REVIEW

**Comment by levimeahan** [comment]
What is the intent of the adminPanel changes?

File: classes/User.php [hunk 1]
```diff
@@ -254,6 +254,7 @@ public function loadData($UPDATE = 2, $remote_view = false): string {
 		}
 		else {
 			$blacklist_json = json_encode($this->blacklist);
+			$this->original_blacklist = $this->blacklist;
```
Comment (levimeahan) [inline classes/User.php:257]: let's revert this, fix merged in https://github.com/levimeahan/shinobi-chronicles/pull/106

**Comment by S1lentcodes** [comment]
> 
> 
> What is the intent of the adminPanel changes?

Removed, they were some fixes to error messages but no worries.

**Review by levimeahan — APPROVED** [review]

**Comment by levimeahan** [comment]
> 
> 
> > What is the intent of the adminPanel changes?
> 
> Removed, they were some fixes to error messages but no worries.

Gotcha, appreciate the error fixes but 3600 lines of changes is a lot to review 😅

**Pull request merged**